File: AlgoData/PrepLab/Assignment3/3.java
import java.io.*;
import java.util.*;

public class 3{
    public static void main(String[] args)throws IOException{
        int nrElements;
        FileReader fr = new FileReader("/Users/Daniel/Documents/AlgoData/PrepLab/Assignment3/3input.txt");
        Scanner scan = new Scanner(fr);
        nrElements = scan.nextInt();

        int[] intArray = new int[nrElements];
        
        for (int i = 0; i < intArray.length; i++){
            nrElements = scan.nextInt();
            intArray[i] = nrElements;
        }
        fr.close();

        for(int i = 0; i < intArray.length; i++){
            System.out.println(intArray[intArray.length - (i+1)]);
        }
    }
}

File: AlgoData/PrepLab/Assignment3/3.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, const char* argv[] ){
    int nrElements, *dynVec, i;
    FILE *fp = fopen("/Users/Daniel/Documents/AlgoData/PrepLab/Assignment3/3input.txt","r");

    fscanf(fp,"%d", &nrElements);
    const int size = nrElements;
    dynVec = (int *) malloc(size*sizeof(int));

    for(i = 0; i < size; i++){
        fscanf(fp,"%d", &nrElements);
        dynVec[i] = nrElements;
    }

    for(i = 0; i < size; i++){
    printf("%d",dynVec[size-(i+1)]);
    printf("\n");
    }

    system("pause");
    return 0;
}